services/src/main/scala/cromwell/pipeline/service/UserService.scala

```diff
@@ -13,38 +13,45 @@ class UserService(userRepository: UserRepository)(implicit executionContext: Exe
   def getUsersByEmail(emailPattern: String): Future[Seq[User]] =
     userRepository.getUsersByEmail(emailPattern)
 
-  def deactivateUserById(userId: UserId): Future[Option[UserNoCredentials]] =
+  def deactivateUserById(userId: UUID): Future[Option[UserNoCredentials]] =
     for {
       _ <- userRepository.deactivateUserById(userId)
       user <- userRepository.getUserById(userId)
     } yield user.map(UserNoCredentials.fromUser)
 
-  def updateUser(userId: String, request: UserUpdateRequest): Future[Int] =
-    userRepository.getUserById(UserId(userId)).flatMap { userOpt =>
-      userOpt.map { user =>
-        userRepository.updateUser(
-          user.copy(email = request.email, firstName = request.firstName, lastName = request.lastName)
-        )
-      } match {
-        case Some(value) => value
-        case None        => Future.failed(new RuntimeException("user with this id doesn't exist"))
-      }
-    }
+  def updateUser(userId: UUID, request: UserUpdateRequest): Future[Int] =
+    userRepository
+      .getUserById(userId)
+      .flatMap(
+        userOpt =>
```

Comment:
I think, we can use pattern matching on the userOpt. I did the same thing on my updateProject method

Reply:
Done